mailbox/tika/src/test/java/org/apache/james/mailbox/tika/TikaExtension.java

```diff
@@ -27,11 +27,7 @@
 import org.junit.jupiter.api.extension.ParameterResolver;
 
 public class TikaExtension implements AfterAllCallback, BeforeAllCallback, ParameterResolver {
-    private final TikaContainer tika;
-
-    public TikaExtension() {
-        this.tika = new TikaContainer();
-    }
+    private final TikaContainer tika = TikaContainerSingleton.INSTANCE;
```

Comment:
Stop calling start / stop in beforeAll / afterAll

Reply:
I totally get that, but I got confused when looking in other extensions (like Cassandra or Elasticsearch) where we totally do that... So we agree this behavior is wrong, and we might need to change it at some point for other docker extensions using singletons as well?